Samples/Common/DockerUtils.cs

```diff
@@ -0,0 +1,408 @@
+                        ".azuredocker",
+                        true);
+
+                sshShell.Upload(Encoding.ASCII.GetBytes(CREATE_OPENSSL_TLS_CERTS_FOR_UBUNTU.Replace("HOST_IP", dockerHostIP)),
+                        "CREATE_OPENSSL_TLS_CERTS_FOR_UBUNTU.sh",
+                        ".azuredocker",
+                        true);
+                sshShell.Upload(Encoding.ASCII.GetBytes(INSTALL_DOCKER_TLS_CERTS_FOR_UBUNTU),
+                        "INSTALL_DOCKER_TLS_CERTS_FOR_UBUNTU.sh",
+                        ".azuredocker",
+                        true);
+                sshShell.Upload(Encoding.ASCII.GetBytes(DEFAULT_DOCKERD_CONFIG_TLS_ENABLED),
+                        "dockerd_tls.config",
+                        ".azuredocker",
+                        true);
+                sshShell.Upload(Encoding.ASCII.GetBytes(CREATE_DEFAULT_DOCKERD_OPTS_TLS_ENABLED),
+                        "CREATE_DEFAULT_DOCKERD_OPTS_TLS_ENABLED.sh",
+                        ".azuredocker",
+                        true);
+                sshShell.Upload(Encoding.ASCII.GetBytes(DEFAULT_DOCKERD_CONFIG_TLS_DISABLED),
+                        "dockerd_notls.config",
+                        ".azuredocker",
+                        true);
+                sshShell.Upload(Encoding.ASCII.GetBytes(CREATE_DEFAULT_DOCKERD_OPTS_TLS_DISABLED),
+                        "CREATE_DEFAULT_DOCKERD_OPTS_TLS_DISABLED.sh",
+                        ".azuredocker",
+                        true);
+
+                Utilities.Log("Trying to install Docker host at: " + dockerHostIP);
+                string output = sshShell.ExecuteCommand("chmod +x ~/.azuredocker/INSTALL_DOCKER_FOR_UBUNTU_SERVER_16_04_LTS.sh");
+                Utilities.Log(output);
+                output = sshShell.ExecuteCommand("bash -c ~/.azuredocker/INSTALL_DOCKER_FOR_UBUNTU_SERVER_16_04_LTS.sh");
+                Utilities.Log(output);
+
+                Utilities.Log("Trying to create OPENSSL certificates");
+                output = sshShell.ExecuteCommand("chmod +x ~/.azuredocker/CREATE_OPENSSL_TLS_CERTS_FOR_UBUNTU.sh");
+                Utilities.Log(output);
+                output = sshShell.ExecuteCommand("bash -c ~/.azuredocker/CREATE_OPENSSL_TLS_CERTS_FOR_UBUNTU.sh");
+                Utilities.Log(output);
+
+                Utilities.Log("Trying to install TLS certificates");
+
+                output = sshShell.ExecuteCommand("chmod +x ~/.azuredocker/INSTALL_DOCKER_TLS_CERTS_FOR_UBUNTU.sh");
+                Utilities.Log(output);
+                output = sshShell.ExecuteCommand("bash -c ~/.azuredocker/INSTALL_DOCKER_TLS_CERTS_FOR_UBUNTU.sh");
+                Utilities.Log(output);
+                Utilities.Log("Download Docker client TLS certificates from: " + dockerHostIP);
+                //keyPemContent = sshShell.Download("key.pem", ".azuredocker/tls", true);
+                //certPemContent = sshShell.Download("cert.pem", ".azuredocker/tls", true);
```

Comment:
these as well. No need to have commented chunks of code. Please clean it everywhere.